CONTRIBUTING.md

```diff
@@ -1 +1,20 @@
+Thank you very much for taking the time to contribute! 
```

Comment:
```suggestion
# Contributing to Elixir RDM-toolkit

Thank you very much for taking the time to contribute! 
```

Reply:
I added a top title. Since this is only to live on github, we can use H1 titles :)